configuration/messageproperties/messages_en.properties

```
@@ -1280,6 +1280,8 @@ pihcore.visitNote.plan.medication.liver.disease=Medications for Liver Disease
 pihcore.visitNote.plan.medication.heart.failure=Medications for Heart Failure
 pihcore.ncd.htn.or.diabetes=Hypertension / Diabetes
 
+pihcore.hiv.legacyDataNotMigrated=LEGACY DATA from HIV EMR, not fully migrated
```

Maybe we can elaborate on what this means for the end user?

```suggestion
pihcore.hiv.legacyDataNotMigrated=WARNING: This data comes from an old form in the old HIV EMR. Some data which may have been filled in on the old form might not appear here.
```

---

Yeah, that was really just a starting message... wanted Dave and Debbie and others to take a look and determine what they really want it to say.  Happy to go with your approach as a first pass though...